src/components/ProgramDiv.tsx

```diff
@@ -6,12 +6,12 @@ type Props = {
 
 export const ProgramDiv = ({ time, headline, description }: Props) => {
   return (
-    <div className="flex content-start gap-10 my-6">
-      <p className="font-bold">{time}</p>
-      <div>
-        <p className="font-bold">{headline}</p>
-        <p>{description}</p>
+    <div className="flex items-start justify-start">
+      <div className="w-24 font-bold text-2xl font-numbers">{time}</div>
```

Comment:
be aware of the fixed width, it could cause problems while scaling the screen down to mobile devices. I'd rather use some paddings and then maybe a max width